rmc/pipeline/regional_constraint.py

```diff
@@ -63,212 +45,42 @@
-                    context_ht.exome_coverage,
-                    coverage_model,
-                    args.high_cov_cutoff,
-                )
-            )
-
-            # TODO: Make this a separate section of code (with its own argument)
-            if not args.skip_calc_oe:
-                logger.info(
-                    "Adding coverage correction to mutation rate probabilities..."
-                )
-                context_ht = context_ht.annotate(
-                    raw_mu_snp=context_ht.mu_snp,
-                    mu_snp=context_ht.mu_snp
-                    * get_coverage_correction_expr(
-                        context_ht.exome_coverage, context_ht.coverage_model
-                    ),
-                )
-
-                logger.info(
-                    "Creating autosomes + chrX PAR, chrX non-PAR, and chrY non-PAR HT"
-                    " versions..."
-                )
-                context_auto_ht = filter_to_region_type(context_ht, "autosomes")
-                context_x_ht = filter_to_region_type(context_ht, "chrX")
-                context_y_ht = filter_to_region_type(context_ht, "chrY")
-
-                logger.info("Calculating expected values per transcript...")
-                exp_ht = calculate_exp_per_transcript(
-                    context_auto_ht,
-                    locus_type="autosomes",
-                    groupings=GROUPINGS,
-                )
-                exp_x_ht = calculate_exp_per_transcript(
-                    context_x_ht,
-                    locus_type="X",
-                    groupings=GROUPINGS,
-                )
-                exp_y_ht = calculate_exp_per_transcript(
-                    context_y_ht,
-                    locus_type="Y",
-                    groupings=GROUPINGS,
-                )
-                exp_ht = exp_ht.union(exp_x_ht).union(exp_y_ht)
-
-                logger.info(
-                    "Fixing expected values for genes that span PAR and nonPAR"
-                    " regions..."
-                )
-                # Adding a sum here to make sure that genes like XG that span PAR/nonPAR regions
-                # have correct total expected values
-                exp_ht = exp_ht.group_by(transcript=exp_ht.transcript).aggregate(
-                    section_exp=hl.agg.sum(exp_ht.expected),
-                    section_mu=hl.agg.sum(exp_ht.mu_agg),
-                )
-                # TODO: Write exp HT here
-
-                logger.info(
-                    "Aggregating total observed variant counts per transcript..."
-                )
-                obs_ht = calculate_observed(exome_ht)
-                # TODO: Write obs HT here
-
+            logger.info("Creating constraint prep HT...")
+            if (
+                not args.prep_missense
+                and not args.prep_nonsense
+                and not args.prep_synonymous
+            ):
+                create_constraint_prep_ht(filter_csq=False, overwrite=args.overwrite)
```

Comment:
```suggestion
                create_constraint_prep_ht(overwrite=args.overwrite)
```
(if we change `filter_csq` to be a set)